roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,401 @@
+other validator in at most 2 hops, providing redundancy.
+
+Propagation follows these rules:
+
+- For groups that we are in, receive from nobody and send to our X/Y peers.
+- For groups that we are not part of:
+  - We receive from any validator in the group we share a slice with and send to
+    the corresponding X/Y slice in the other dimension.
+  - For any validators we don't share a slice with, we receive from the nodes
+    which share a slice with them.
+
+### Seconding Limit
+
+The seconding limit is a per-validator limit. Before asynchronous backing, we
+had a rule that every validator was only allowed to second one candidate per
+relay parent. With asynchronous backing, we have a 'maximum depth' which makes
+it possible to second multiple candidates per relay parent. The seconding limit
+is set to `max depth + ` to set an upper bound on candidates entering the
+system.
+
+## Candidates Module
+
+The candidates module provides a tracker for all known candidates in the view,
+and whether they are confirmed or not. What is a confirmed candidate? The first
+time a validator gets an announcement for an unknown candidate, it will send a
+request for the candidate. Upon receiving a response and validating it (see
+`UnhandledResponse::validate_response`), will mark the candidate as confirmed.
+
+## Requests Module
+
+The requests module provides a manager for pending requests for candidate data,
+as well as pending responses. See "Request/Response Protocol" for a high-level
+description of the flow. See module-docs for full details.
+
+## Glossary
+
+- **Acknowledgement:** A notification that is sent to a validator that already
+  has the candidate, to inform them that the sending node knows the candidate.
+- **Announcement:** A notification of a backed candidate being known by the
+- **Attestation:** See "Statement".
+  sending node. Is a full manifest and initiates manifest exchange.
+- **Manifest:** A message about a known backed candidate, along with a
+  description of the statements backing it. See "Manifests" section.
+- **Peer:** Another validator that a validator is connected to.
+- **Request/response:** A protocol used to lazily request and receive heavy
+  candidate data when needed.
+- **Reputation:** Tracks reputation of peers. Applies annoyance cost and good
+  behavior benefits.
+- **Statement:** Signed statements that can be made about parachain candidates.
+  - **Seconded:** Proposal of a parachain candidate. Implicit validity vote.
+  - **Valid:** States that a parachain candidate is valid.
+- **Target:** Target validator to send a statement to.
+- **View:** Current knowledge of the chain state.
+  - **Explicit view** / **immediate view**
+    - The view a peer has of the relay chain heads and highest finalized block.
+  - **Implicit view**
+    - Derived from the immediate view. Composed of active leaves and minimum
+      relay-parents allowed for candidates of various parachains at those
+      leaves.
```

Comment:
While I understand the meaning of this, I think as per "guide" it should be explained like
> Composed of active leaves and their ancestries which are allowed for building parachain blocks in the context of.

Or maybe some better wording

Reply:
Yeah I guessed that my wording wouldn't be much better. Let's disregard then